manager/src/lib.rs

```diff
@@ -616,7 +602,25 @@ impl CronManager {
 
         // only skip scheduling if user didnt intend
         let current_slot = self.get_slot_id(None);
-        let call_balance_used = self.task_balance_uses(&task);
+        // let call_balance_used = self.task_balance_uses(&task);
+        let call_balance_used = u128::from(env::used_gas());
```

Comment:
So yeah, here we should multiply the value by something that can be updated via the `update_settings` function. The default gas price again is `100000000`

Reply:
I'll add that as a setting, i saw some early blocks yesterday that had a larger gas price, but it was only from 1 validator.. anyway, setting updated via DAO will be :100: